### ISSUE TITLE
cooliris.swf is still looking for some files hosted on cooliris.com

### ISSUE THREAD

**Issue by JimOL** [cooliris/embed-wall#4]
With all respect, it is not a good answer to say that Yahoo bought Cooliris and therefore you are not responsible for cooliris.swf. I have invested several years worth of web albums created with cooliris and now although I have downloaded cooliris.swf to my web host, none of the albums show up because coolliris.swf is looking for crossdomain.xml on your site. It should be easy for your developers who created this complex software to open it in Flash and change it so that cooliris.swf is looking for crossdowmain.xml on a relative path.

Thanks

**Comment by librarymark** [comment]
I agree - if you are going to give this away, give ALL of it away, source included. That we we can fix issues like the files that the .swf is still searching for that don't exist any more.

**Comment by byman64** [comment]
# Same issue here :-(
Embeded url in the cooliris.swf file search for :
http://apps.cooliris.com/embed/images/c2_logo_black.png
https://e.cooliris.com/embed/v/900/embed.js
http://apps.cooliris.com/crossdomain.xml
http://apps.cooliris.com/embed/images/sharing2.png

Some has the source...it would be easy make some changes...

**Comment by librarymark** [comment]
Not a whole lot of point now that Flash is going to go away in a few years.
I wish there was an HTML5 replacement. My users love the image wall, and
there is nothing out there comparable. Yahoo did us no favors by swallowing
up a great product.

**Comment by OLeary** [comment]
I have changed my web album to [Lightbox](http://lokeshdhakar.com/projects/lightbox2/). I found this javascript library easy to implement and it has a lot of options. Most importantly, it will not suddenly stop working like cooliris.

**Comment by byman64** [comment]
I would like just update a my old project, I already migrated to phoswipe.com, another great product.
[an example from my site](https://www.byman.it/site/index.php?option=com_content&view=article&id=284:sky-and-clouds&catid=48:photos&Itemid=115)

But there is not another coolires wall3d valid like it :-(
